tests_integration/__tests__/ignore-relative-path.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const path = require("path");
+const runPrettier = require("../runPrettier");
+
+describe("support relative paths", () => {
+  runPrettier("cli/ignore-relative-path", [
+    path.resolve(__dirname, "../cli/ignore-relative-path/level1/level2/level3/file.js"),
+    path.resolve(__dirname, "../cli/ignore-relative-path/anotherfile.js"),
```

Comment:
These should be relative paths, `path.resolve()` will make them absolute.

Reply:
Yes I know, I just wanted to push the changes & will clean it up the whole PR in an hour or two 😊

Reply:
Let me know if you're ready. 👍